src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
-  margin: 0;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+
+.product-card {
+  font-family: Roboto, Arial, Helvetica, 'Segoe UI', sans-serif, system-ui;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
+
+  &__page {
+    margin: 0;
+  }
+
+  &__all {
+    box-sizing: border-box;
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    background: $white-accent;
+    border-radius: 5px;
+  }
```

Comment:
The BEM element '__all' seems to be a bit ambiguous in terms of naming. Consider renaming it to something more descriptive of its function or content, adhering to the BEM naming conventions and making it clear what the element represents.